configure.py

```diff
@@ -382,8 +384,8 @@ def search_system_path(file_name):
         cflags.append('-D_LARGE_FILES')
     if platform.is_zos():
         cflags += ['-D_POSIX_SOURCE', '-D_POSIX_C_SOURCE=200112L', '-D_XOPEN_SOURCE_EXTENDED=1', '-D_UNIX03_THREADS']
-        cflags.remove('-pipe')
-        cflags.remove('-std=c++11')
+        # cflags.remove('-pipe')
+        # cflags.remove('-std=c++11')
```

Comment:
I would leave this flag here to ensure that the Ninja sources always exclusively use C++11 constructs, instead of whatever the compiler supports by default (which could be C++14 or anything more recent).